### PR TITLE
ENT-10971: new file and command component usage across markdown in getting started section

### PR DESCRIPTION
**Pull request by mineralsfree**
Ticket: ENT-10971
Changelog: None

### CODE REVIEW

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/474/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/474/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-474/output/_site/

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/475/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/475/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-475/output/_site/

File: getting-started/installation/general-installation.markdown [hunk 1]
```diff
@@ -48,13 +48,16 @@ Run the bootstrap command, **first** on the policy server:
 1. Find the IP address of your Policy Server:
 
 
-	    $ ifconfig
+```command
+        ifconfig
```
Comment (olehermanse) [inline getting-started/installation/general-installation.markdown:52]:
```suggestion
ifconfig
```

File: getting-started/installation/general-installation.markdown [hunk 2]
```diff
@@ -48,13 +48,16 @@ Run the bootstrap command, **first** on the policy server:
...
+```
 
 
 2. Run the bootstrap command:
 
-
-        $ sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of policy server>
+```command
+         sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of policy server>
```
Comment (olehermanse) [inline getting-started/installation/general-installation.markdown:59]:
```suggestion
sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of policy server>
```

File: getting-started/installation/general-installation/installation-coreos.markdown [hunk 3]
```diff
@@ -17,14 +17,14 @@ Download the file-system image package for CoreOS from the [Enterprise Downloads
...
+    ```command
+    tar xvf cfengine-nova-{{site.cfengine.branch}}.{{site.cfengine.latest_patch_release}}-{{site.cfengine.latest_package_build}}.x86_64.fs-img.pkg.tar.gz
     ```
 
 2. On the CoreOS Host, run the install script:
 
-    ```console
-    core@coreos ~ $ sudo ./cfengine-nova-{{site.cfengine.branch}}.{{site.cfengine.latest_patch_release}}-{{site.cfengine.latest_package_build}}.x86_64.fs-img.pkg/install.sh
+    ```command
+     sudo ./cfengine-nova-{{site.cfengine.branch}}.{{site.cfengine.latest_patch_release}}-{{site.cfengine.latest_package_build}}.x86_64.fs-img.pkg/install.sh
```
Comment (olehermanse) [inline getting-started/installation/general-installation/installation-coreos.markdown:27]:
```suggestion
    sudo ./cfengine-nova-{{site.cfengine.branch}}.{{site.cfengine.latest_patch_release}}-{{site.cfengine.latest_package_build}}.x86_64.fs-img.pkg/install.sh
```

File: getting-started/installation/general-installation/installation-coreos.markdown [hunk 4]
```diff
@@ -33,8 +33,8 @@ Note: Install actions logged to `/var/log/CFEngine-Install.log`.
 
 Run the bootstrap command:
 
-```console
-core@coreos ~ $ sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of the Policy Server>
+```commmand
+ sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of the Policy Server>
```
Comment (olehermanse) [inline getting-started/installation/general-installation/installation-coreos.markdown:37]:
```suggestion
sudo /var/cfengine/bin/cf-agent --bootstrap <IP address of the Policy Server>
```

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/476/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/476/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-476/output/_site/

**Review by olehermanse — APPROVED** [review]

**Comment by olehermanse** [comment]
Cherry-pick: https://github.com/cfengine/documentation/pull/3171

**Pull request merged**